assets/js/screens/Work/List.jsx

```diff
@@ -1,53 +1,103 @@
-import React from "react";
+import React, { useState } from "react";
 import Layout from "../Layout";
-import SearchResultItem from "../../components/Search/ResultItem";
 import { ReactiveList } from "@appbaseio/reactivesearch";
-import { IIIFProvider } from "../../components/IIIF/IIIFProvider";
+import { IIIFProvider, IIIFContext } from "../../components/IIIF/IIIFProvider";
+import WorkCardItem from "../../components/Work/UIWorkCardItem";
+import WorkListItem from "../../components/Work/UIWorkListItem";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 const ScreensWorkList = () => {
+  const [isListView, setIsListView] = useState(false);
+
+  const getWorkItem = (res) => {
+    return {
+      id: res._id,
+      title: res.title,
+      updatedAt: res.modified_date,
+      representativeImage: { id: res.representative_file_set_id, value: null },
+      manifestUrl: res.iiif_manifest,
+      published: res.published,
+      visibility: res.visibility.id
+        ? res.visibility
+        : { id: res.visibility, label: res.visibility },
```

Comment:
@kdid @mbklein  Is there someplace we can see/track which properties will always come along for the ride on an Elasticsearch result data structure?   Via Kibana, or is there some schema definition file someplace that details this?

@katdivyareddy10 For example here, some of my results (could be just one), doesn't have `visibility`, so it's crashing.  Why don't we wrap the entire object preparation in a `try {} catch {}` to get around crashing, then individually guard against the fields being present?    

Here's we should check for `res.visibility` on line 20, then let the logic flow off that.

Reply:
Also, this is my doing from the past, but could you make a guard against a `false` or "" value coming into this helpers file?  As long as we're in it...

![image](https://user-images.githubusercontent.com/3020266/81306088-b2ca4e00-9044-11ea-95e4-025567faf5fa.png)

Either give a default assignment in the parameter (`visibility = ""`, which might not work), or on the first line of the function:
```
if (!visibility) return;
```

Reply:
We had to remove visibility briefly while we redo the backend. I'm not sure that there is a way to see the mappings via Kibana.

You could look at the code where we do the indexing to see the current state:
https://github.com/nulib/meadow/blob/deploy/staging/lib/meadow/data/schemas/work.ex#L82

We can try to be better about giving a heads up.

Reply:
or we could index static values in the Elasticsearch index for now if it would help...
like "visibility": "NOT IMPLEMENTED"

Reply:
@kdid Gotcha.  Actually the last option, just including the stubbed property of whatever datatype it's going to be, would be most helpful.  

So if it's a `string`, then yea ` { visibility: "WHATEVER" }` is fine.

If it's an object like in the mock data, then `{ visibility: { label: "", id: "" } }` would be most helpful.

We'd like to find a balance between defensive coding, and not creating a bunch of bloat that eventually won't be needed.

Hopefully this makes sense?   Lemme know any questions.  Thanks!

@katdivyareddy10  and myself are both looking at this now.